pkg/config/images.go

```diff
@@ -39,12 +21,10 @@ func openShiftVersion(imageVersion string) (string, error) {
 func (g *simpleGenerator) selectNodeImage(cs *api.OpenShiftManagedCluster) {
```

Comment:
I think it's worth renaming this to denote that we are only doing defaulting now. `setDefaults` or something across those lines.

Reply:
we selecting nodeImage version here explicitly, so I think name corresponds ok :)

Reply:
"select" is pretty vague, what you actually do is set defaults in case necessary values are not set by the merged plugin template.

Reply:
I think we dont need this anymore. We pass those from plugin config now